examples/2d_mean_flow.ipynb

```diff
@@ -0,0 +1,419 @@
+        "            nn.Linear(input_dim + 2*time_dim, hidden_dim),\n",
+        "            Swish(),\n",
+        "            *[nn.Sequential(\n",
+        "                nn.Linear(hidden_dim, hidden_dim),\n",
+        "                Swish(),\n",
+        "            ) for _ in range(6)],\n",
+        "            nn.Linear(hidden_dim, input_dim),\n",
+        "            )\n",
+        "\n",
+        "    def forward(self, x: Tensor, r: Tensor, t: Tensor) -> Tensor:\n",
+        "        sz = x.size()\n",
+        "        x = x.reshape(-1, self.input_dim)\n",
+        "        t = t.reshape(-1, self.time_dim).float()\n",
+        "        r = r.reshape(-1, self.time_dim).float()\n",
+        "\n",
+        "        t = t.reshape(-1, 1).expand(x.shape[0], 1)\n",
+        "        r = r.reshape(-1, 1).expand(x.shape[0], 1)\n",
+        "        h = torch.cat([x, r, t], dim=1)\n",
+        "        output = self.main(h)\n",
+        "\n",
+        "        return output.reshape(*sz)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "uk-Mw-9WSLkM"
+      },
+      "source": [
+        "## Train MeanFlow model"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "cp24XAWWSLkM",
+        "outputId": "7e5dfb0c-0c9e-40d9-bfe6-b2ab514f418f"
+      },
+      "outputs": [],
+      "source": [
+        "# training arguments\n",
+        "lr = 3e-4\n",
+        "batch_size = 4096\n",
+        "iterations = 40001\n",
+        "print_every = 2000\n",
+        "hidden_dim = 1024\n",
+        "\n",
+        "# average velocity field model init\n",
+        "avf = MLP(input_dim=2, time_dim=1, hidden_dim=hidden_dim).to(device)\n",
+        "# avf = torch.nn.DataParallel(avf, device_ids=list(range(torch.cuda.device_count())))\n",
+        "\n",
+        "# instantiate an affine path object\n",
+        "path = AffineProbPath(scheduler=CondOTScheduler())\n",
+        "\n",
+        "# init optimizer\n",
+        "optim = torch.optim.Adam(avf.parameters(), lr=lr)\n",
+        "\n",
+        "# train\n",
+        "start_time = time.time()\n",
+        "for i in range(iterations):\n",
+        "    optim.zero_grad()\n",
+        "\n",
+        "    # sample data (user's responsibility): in this case, (X_0,X_1) ~ pi(X_0,X_1) = q(X_1)N(X_0|0,I)\n",
+        "    x_0 = inf_train_gen(batch_size=batch_size, device=device) # sample data\n",
+        "    x_1 = torch.randn_like(x_0).to(device)\n",
+        "\n",
+        "    # sample two time points from a logit-normal distribution\n",
+        "    r = torch.sigmoid(torch.randn(x_1.shape[0], device=device) - 0.4)\n",
+        "    t = torch.sigmoid(torch.randn(x_1.shape[0], device=device) - 0.4) # mean -0.4, var 1\n",
+        "\n",
+        "    # set r = t for 75% of the batch\n",
```

Comment:
I believe the Mean Flows paper says "The ratio of sampling r != t is 75%". In which case, shouldn't `r = t` 25% of the time?